examples/aerial_atomic_swap.py

```diff
@@ -59,16 +60,12 @@ def main():
     print(f"Bob   balance {bob_balance}")
 
     if alice_balance < (10**18):
```

Comment:
Change these if conditions to while loops that keep requesting funds until the amount condition is satisfied.
Apply below as well.